client/client.go

```diff
@@ -247,6 +247,12 @@ func (cli *Client) UpdateClientVersion(v string) {
 
 }
 
+// ClientHost returns the host associated with this instance of the Client.
+// This operation doesn't acquire a mutex.
+func (cli *Client) ClientHost() string {
```

Comment:
I guess this was named for consistency with `ClientVersion`, but `Host` would be better, I think.
(Or `DaemonHost`? )